rrrspec-client/lib/rrrspec/client/support.rb

```diff
@@ -40,7 +40,26 @@ def start_taskset(conf, rsync_name)
         return taskset
       end
 
-      def show_result(taskset, verbose=false)
+      def show_result(taskset, verbose=false, show_errors=false)
+        if show_errors
```

Comment:
I'm not fully convinced. Isn't it making sense to put this in the last part to see the failing specs if you're running this in a console?